scripts/commands/updateApiDocs.ts

```diff
@@ -154,18 +180,18 @@ zxMain(async () => {
     await downloadHtml({ baseUrl: pkg.baseUrl, initialUrls: pkg.initialUrls, destination });
   }
 
-  // Sphinx html to markdown
-  console.log('Deleting existing markdowns');
-  await $`rm -rf ${getRoot()}/docs/api/`;
```

Comment:
This was deleting our new section `/api/migration-guides`, which is not auto-generated. We need to be more granular.